util/winutil/winutil_notwindows.go

```diff
@@ -22,3 +22,13 @@ func GetRegString(name, defval string) string { return defval }
 // This function will only work on GOOS=windows. Trying to run it on any other
 // OS will always return the default value.
 func GetRegInteger(name string, defval uint64) uint64 { return defval }
+
+// IsSIDValidPrincipal determines whether the SID contained in uid represents a
```

Comment:
only document this once in a portable file, otherwise you'll either have to mirror it in two files and keep it in sync or it won't show up in godoc for people on the 'wrong' platform(s).

common pattern is put the docs on the exported func in a portable file and then delegate to an unexported func/method in platform-specific files.

Reply:
Got it. I've reorganized the affected functions according to that convention.